main.py

```diff
@@ -0,0 +1,60 @@
+from typing import List
+
+import crud
+import models
+import schemas
+
+from database import SessionLocal
+from database import engine
+from fastapi import Depends
+from fastapi import FastAPI
+from fastapi import HTTPException
+from sqlalchemy.orm import Session
+
+models.Base.metadata.create_all(bind=engine)
+
+app = FastAPI()
+
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.post("/authors/", response_model=schemas.Author)
+def create_author(author: schemas.AuthorCreate, db: Session = Depends(get_db)):
+    return crud.create_author(db=db, author=author)
```

Comment:
Author's name is a unique field and when you create an author with an existing name you gonna get Internal server error, so you should handle it